02_housing/2a_affordable_available_place.R

```diff
@@ -110,7 +110,7 @@ available_2022 <- available_2022 %>%
 # (4) calculate the number of affordable and available units at each income level per 100 households
 # this is the number of units that are rented to people at or below each income threshold
 # plus the number of vacant units at that income threshold over the total number
-# of households in each income bracket
```

Comment:
Now that we resolved the issue on the 1a script this calculation now seems correct. However,
1. My understanding (from Claudia) is that these variables should be named share_ rather than rate_ to be consistent with the other final files
2. They should not be multiplied by 100, again to be consistent with the other final files